cclib/bridge/cclib2horton.py

```diff
@@ -53,7 +55,32 @@ def makehorton(ccdat):
     attributes = {}
```

Comment:
So I'm a little confused, because IIUC we want to pass in attribute to this function but we are passing a ccData object. What will you do in the end? For your project, I don't think it matters much.

For long term standardization, created #890

Reply:
@langner I've modified the function so accept `ccData` object after your suggestion on PR #873. Modifying from this to a function that asks for the attributes directly should not be too difficult, if that is the way we will be standardizing the bridge functions.

Reply:
This is tricky because this bridge can take many more attributes than all the other existing ones. It might be unreasonable to ask users to come up with all of these function arguments and have most default to `None` (or require explicitly setting to `None`). I agree that we shouldn't fuss over it and any changes we want can be made later.

Reply:
Sounds good.